site/slices/contact-us-slice/form/index.js

```diff
@@ -15,6 +15,23 @@ class Form extends Component {
     };
   }
 
+  componentDidMount = () => {
+    const { stateNavigator } = this.context;
+    stateNavigator.onNavigate(this.focusText);
+  }
+
+  componentWillUnmount = () => {
+    const { stateNavigator } = this.context;
+    stateNavigator.offNavigate(this.focusText);
+  }
```

Comment:
Hmm it'd be nice if you didn't need to know about `stateNavigator` from this component at all, but now you're not just depending on the query string it's probably not realistic to do another way. Ignore me.